Example-Crypto-REXX/README.md

```diff
@@ -0,0 +1,104 @@
+      - **required**
+      - *outputDataSet* is the name of an existing partitioned data set
+      - if the query is successful, a member containing the crypto
+        information in a CSV format will be stored into the specified data set: *outputDataSet(memberName)* where the name of data set member is:
+        - LOCAL, if the LOCAL CPC was used
+        - CPCname, if a specific CPC was specified via the -C option
+  - *-C CPCname*
+      - **optional**
+      - *CPCname* is the name of the CPC whose crypto info will be retrieved
+      - **default if not provided:** LOCAL CPC
+  - *-S Status*
+      - **optional**
+      - *Status* is the desired LPAR Status of the profiles to query 
+      - Valid values:          
+        - operating - query profiles for active/operating images  
+        - not-operating - query profiles for active/not operating images                                                   
+        - not-activated - query profiles for not activated images 
+      - **default if not provided or if not valid:** operating                                               
+  - *-I*
+      - **optional**
+      - indicates this exec is being run in an ISV REXX environment
+      - **default if not set:** TSO/E REXX environment
+  - *–V*
+      - **optional**
+      - indicates additional tracing associated with JSON parsing should be turned on
+      - **default if not set:** tracing excludes JSON specific tracing
+
+**sample invocation in TSO:**
+<br>RXCRYPT1 has been copied into data set HWI.HWIREST.REXX and HWI.RXCRYPTO.OUTPUT
+    has been allocated as RECFM=VB, LRECL=800
+```
+ex 'HWI.HWIREST.REXX(RXCRYPT1)' '-D HWI.RXCRYPTO.OUTPUT'
+```
+ - exec is running in a TSO/E rexx environment and will query the LOCAL CPC
+```
+ex 'HWI.HWIREST.REXX(RXCRYPT1)' '-D HWI.RXCRYPTO.OUTPUT -C CPC1'
+```
+ - exec is running in a TSO/E rexx environment and will query CPC CPC1
+ ```
+ex 'HWI.HWIREST.REXX(RXCRYPT1)' '-D HWI.RXCRYPTO.OUTPUT -C CPC1 -S not-activated'
+```
+ - exec is running in a TSO/E rexx environment, will query CPC CPC1 and query info for LPARs which are in not-activated status
+
+**sample batch invocation via JCL:**
+<br>RXCRYPT1 has been copied into data set HWI.HWIREST.REXX and HWI.RXCRYPTO.OUTPUT
+    has been allocated as RECFM=VB, LRECL=800
+
+```
+ //RXCRYPT1 JOB ,
+ // CLASS=J,NOTIFY=&SYSUID,MSGLEVEL=1,
+ //  MSGCLASS=H,REGION=0M,TIME=1440
+ //STEP1    EXEC PGM=IKJEFT01,DYNAMNBR=20
+ //SYSUDUMP DD SYSOUT=(H,,STD)
+ //SYSTSPRT DD SYSOUT=(H,,STD)
+ //SYSTSIN  DD * 
+ PROFILE NOPREFIX
+ EX 'HWI.HWIREST.REXX(RXCRYPT1)' -
+ '-D HWI.RXCRYPTO.OUTPUT -C CPC1'
+ /*
+ ```
+ - exec is running in a TSO/E rexx environment and will query CPC CPC1
+
+## Generated Output
+
+ ![Sample crypto result](images/SampleCryptoResult.png)
+ - If no query results show up, ensure that
+  1. Your USER ID has the appropriate access level to the FACILITY Class profile associated with that CPC and images, see **System Prep Work** above.    
```

Comment:
change images to LPARs and Image Activation Profiles

Reply:
changed images to image name, access to the image activation profile is at the CPC level